src/main/java/org/yardstickframework/runners/context/DockerContext.java

```diff
@@ -0,0 +1,312 @@
+package org.yardstickframework.runners.context;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Map;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.constructor.Constructor;
+
+/**
+ * Docker context.
+ */
+public class DockerContext {
+    /** */
+    private static final Logger LOG = LogManager.getLogger(DockerContext.class);
+
+    /** */
+    private String serverDockerfilePath = "config/docker/Dockerfile-server";
+
+    /** */
+    private String driverDockerfilePath = "config/docker/Dockerfile-driver";
+
+    /** */
+    private String serverImageName = "yardstickserver";
+
+    /** */
+    private String driverImageName = "yardstickdriver";
+
+    /** */
+    private boolean rebuildImagesIfExist;
+
+    /** */
+    private Map<String, Boolean> removeImagesFlags;
+
+    /** */
+    private String dockerBuildCmd;
+
+    /** */
+    private String dockerRunCmd;
+
+    /** */
+    private String startContCmd;
+
+    /** */
+    private Map<String, Boolean> removeContainersFlags;
+
+    /** */
+    private List<String> containersToRemove;
+
+    /** */
+    private String serverDockerJavaHome;
+
+    /** */
+    private String driverDockerJavaHome;
+
+    /**
+     *
+     * @return {@code String} server dockerfile path.
+     */
+    public String getServerDockerfilePath() {
+        return serverDockerfilePath;
+    }
+
+    /**
+     *
+     * @param serverDockerfilePath server dockerfile path.
+     */
+    public void setServerDockerfilePath(String serverDockerfilePath) {
+        this.serverDockerfilePath = serverDockerfilePath;
+    }
+
+    /**
+     *
+     * @return {@code String} driver dockerfile path.
+     */
+    public String getDriverDockerfilePath() {
```

Comment:
This methods can be merged into one to avoid code like this

String docFilePath = nodeType == NodeType.SERVER ?
                runCtx.resolveRemotePath(dockerCtx.getServerDockerfilePath()) :
                runCtx.resolveRemotePath(dockerCtx.getDriverDockerfilePath())

You can use String docFilePath = runCtx.resolveRemotePath(dockerCtx.getDockerfilePath(nodeType))

    public String getDockerfilePath(NodeType nodeType) {
        if (nodeType == SERVER)
             return serverfilePath;
....
    }